datadog/resource_datadog_dashboard.go

```diff
@@ -2355,10 +2355,12 @@ func getManageStatusDefinitionSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Optional: true,
 		},
+		// The count param is deprecated
```

Comment:
Could you please add the `Deprecated` attribute to these two? https://godoc.org/github.com/hashicorp/terraform/helper/schema#Schema